managed/src/main/java/com/yugabyte/yw/models/TaskInfo.java

```diff
@@ -273,7 +273,7 @@ public double getPercentCompleted() {
       .eq("parent_uuid", getTaskUUID())
       .findCount();
     if (numSubtasks == 0) {
-      return 0.0;
+      return 100.0;
```

Comment:
why this change?

Reply:
@jaydeepkumara https://github.com/yugabyte/yugabyte-db/pull/7864#discussion_r622530835